integrations.yml

```diff
@@ -438,3 +441,8 @@ integrations:
     url: "https://marker.io/basecamp"
     image: extras/marker-io.png
     category: 5
+  - title: "Monkedo"
+    description: "The Ultimate App for Integration and Automation. Seamlessly connect your favorite apps, automate tasks with ease, and create custom applications without coding."
+    url: "https://monkedo.com/integrations/app/basecamp"
+    image: extras/monkedo.png
+    category: 10
```

Comment:
Can you change the category to 1 instead of creating a new category? That's the bucket we use when no specific category fits, also Zapier is in the same Bucket.

Reply:
- Thanks for the feedback.